src/main/resources/static/styles.css

```diff
@@ -220,35 +223,42 @@ td{
     align-self: center;
 }
 
-#search-container{
+#search-container {
     width: 10vw;
     margin-left: 10vw;
     height: 2.7vw;
 }
-#search-img{
+
+#search-img {
     margin: 0;
     border-radius: 0;
 }
 
-#name, #login, #email{
+#name, #login, #email {
     margin: 0;
 }
 
-img{
+.profile-img {
+    width: 240px;
+    height: 240px;
+    object-fit: cover;
+}
```

Comment:
This only render as oval on a specific screen size. Since the pixel width and height are different on different screen sizes, the image will render differently on each device. The resizing and scaling should work if you use a view measure instead.  
Like:
```suggestion
.profile-img {
    height: 20vw;
    object-fit: cover;
}
```





This is how it looks for me with width: 240px; and height: 240px;
![image](https://github.com/Win-ther/SpringbootGroupProject/assets/130497295/e457e928-1b0f-43a6-bc6d-ce848ba8399f)

Reply:
The code you suggested gives an oval one, that is what I was trying to avoid by using px to crop the picture instead so it will stay round like it is on GitHub. :)